template/src/containers/App/App.js

```diff
@@ -1,36 +1,28 @@
 import React from 'react';
-import { render } from 'react-dom';
 import { Provider } from 'react-redux';
 import { ReactReduxFirebaseProvider } from 'react-redux-firebase';
 import { createFirestoreInstance } from 'redux-firestore';
 import firebase from 'firebase/app';
 import 'firebase/auth';
 import 'firebase/firestore';
-import createStore from './store';
-import Home from './Home';
-import registerServiceWorker from './registerServiceWorker';
-import * as config from 'config';
+import configureStore from '../../store/createStore';
+import { fbConfig, rfConfig } from '../../config';
+import Router from './Router';
 
 // Initialize Firebase instance
-firebase.initializeApp(config.firebase || process.env.REACT_APP_FIREBASE_CONFIG)
-
-const styles = {
-  fontFamily: 'sans-serif',
-  textAlign: 'center',
-};
-
-const store = createStore()
+firebase.initializeApp(fbConfig || process.env.REACT_APP_FIREBASE_CONFIG)
 
+const store = configureStore()
 
 function App() {
   return (
     <Provider store={store}>
       <ReactReduxFirebaseProvider
         firebase={firebase}
-        config={config.rfConfig}
+        config={rfConfig}
         dispatch={store.dispatch}
         createFirestoreInstance={createFirestoreInstance}>
-        <Home />
+        <Router></Router>
```

Comment:
This could be switched to `<Router />` since there isn't any children being passed. That said, we could just leave this as:

```jsx
<BrowserRouter>{routes}</BrowserRouter>
```

Reply:
Cool, that's a good idea. 👍